pkgs/development/ocaml-modules/pbkdf/default.nix

```
@@ -12,12 +12,13 @@ buildDunePackage rec {
 
   src = fetchFromGitHub {
     owner = "abeaumont";
-    repo = "ocaml-pbkdf";
+    repo = "ocaml-${pname}";
     rev = version;
     sha256 = "sha256-dGi4Vw+7VBpK/NpJ6zdpogm+E6G/oJovXCksJBSmqjI=";
   };
 
```

```suggestion
    minimalOCamlVersion = "4.07";
```
https://github.com/ocaml/opam-repository/blob/master/packages/pbkdf/pbkdf.1.2.0/opam#L15